config/initializers/paperclip_defaults.rb

```diff
@@ -4,8 +4,12 @@
 })
 
 if s3_bucket = ENV['PAPERCLIP_S3_BUCKET'].presence
-  Rails.application.config.paperclip_defaults = {
+  paperclip_defaults = {
     storage: :s3,
     bucket: s3_bucket
   }
+  if s3_host_name = ENV['PAPERCLIP_S3_HOST_NAME'].presence
+    paperclip_defaults[:s3_host_name] = s3_host_name
+  end
```

Comment:
I think we could use:
`paperclip_defaults[:s3_host_name] =  ENV['PAPERCLIP_S3_HOST_NAME'].presence`